code/modules/cargo/packs/emergency.dm

```diff
@@ -44,8 +44,28 @@
 	crate_name = "emergency crate"
 	crate_type = /obj/structure/closet/crate/internals
 
+/datum/supply_pack/emergency/medemergency
+	name = "Emergency Medical Supplies"
+	desc = "Emergency supplies for a front-line medic. Contains two boxes of body bags, a medical HUD, a defib unit, medical belt, toxin bottles, epipens, and several types of medical kits."
+	cost = 3000
```

Comment:
oooh this seems like a good "DropPod Only"

Reply:
I went ahead with your suggestion but during testing I found out that drop pods cant be used with defib units. (at least the loaded variant). I was sick of testing at that point, so I boosted the price to 6600 and made it available for regular purchase.